### PR TITLE
Customer address, update password(unfinished)

### PR DESCRIPTION
**Pull request by Marwasel**
Added customer address to create account and update password, using sql lite but no security method or authentication implemented yet.   

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

**Comment by Marwasel** [comment]
I have conflicting files but dont know why and how to solve it.

**Comment by platisd** [comment]
Well, you'll need to figure out why these conflicts exist and solve them. :P

File: CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/StaffLoginPage.java [hunk 1]
```diff
@@ -70,4 +76,10 @@ public void successfulLogin(){     //replace with orders list class when availab
...
             startActivity(intent);
         }
-    }
\ No newline at end of file
+
+
+
+    }
+
+    }
```
Comment (d0nate110) [inline CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/StaffLoginPage.java:84]: The tests point at this line number 84, where the last bracket is, "error: class, interface, or enum expected".

Comment (Marwasel) [reply CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/StaffLoginPage.java]: Thank you! i will change it

**Review by Akuien — APPROVED** [review]

**Pull request merged**